### PR TITLE
cpm requirements update for IOPS

### PR DESCRIPTION
**Pull request by keegoid-nr**
## IOPS

* Add IOPS benchmark values to CPM requirements. The disk can be a source of performance issues for the CPM, especially for write operations. These benchmark values are designed to provide some awareness to disk performance.
* Make units more consistent for Kubernetes
* fix broken link

### CODE REVIEW

**Comment by marcusperezNR** [comment]
These docs have been reviewed by a Synthetics engineer

File: src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx:122]: @keegoid-nr I believe this is an error but can you confirm real quick?

Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx:122]:
```suggestion
            A minimum of 50 GB per host
```

Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx:122]: TIL that Gibabytes are a thing. Ignore this!

**Comment by bradleycamacho** [comment]
@keegoid-nr thanks for the edits! Made a few copyedits and will merge right after the checks complete.

**Review by bradleycamacho — APPROVED** [review]

File: src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx
Comment (keegoid-nr) [reply src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx:122]: One of my goals is to standardize units. We should be using GiB. It is not an error.
https://en.wikipedia.org/wiki/Binary_prefix#Adoption_by_IEC,_NIST_and_ISO

Comment (keegoid-nr) [reply src/content/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms.mdx:122]: Using powers of 1000 becomes less and less accurate as sizes get larger.
https://en.wikipedia.org/wiki/Binary_prefix#Deviation_between_powers_of_1024_and_powers_of_1000

**Pull request merged**
